canonical-web-and-design/ubuntu.com
# Replace usn.ubuntu.com references

## Done

- Redirects to `usn.ubuntu.com` now redirect to `/security/notices`
- Links  to `usn.ubuntu.com` now link to `/security/notices`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Double check the updated redirects work
